pkg/client/clientset/versioned/fake/register.go

```diff
@@ -31,7 +31,6 @@ import (
 
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
-var parameterCodec = runtime.NewParameterCodec(scheme)
```

Comment:
This is a generated file, should be excluded.

Same applies to ` pkg/client`

Reply:
Done.